src/library/masq.js

```diff
@@ -395,14 +418,17 @@ class Masq {
 
       this.sw = swarm(this.hub, swarmOpts)
 
-      this.sw.on('disconnect', async () => {
+      this.swListener = async () => {
```

Comment:
what about renaming the listener to, `onDisconnect` for instance ?

Reply:
ok, rename swListener -> onDisconnect